### PR TITLE
Added more top-level scripts

### PR DESCRIPTION
**Pull request by Corgam**
- Added back the top-level scripts
- Fiixed the problem with angular and docker building having not enough memory

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
     "setup": "npm run setup:frontend && npm run setup:backend",
     "setup:frontend": "cd frontend/ && npm ci",
     "setup:backend": "cd backend/public-api && npm ci && npm run build",
-    "deploy": "docker-compose up --build -d"
+    "deploy": "docker-compose up --build -d",
+    "deploy:backend": "docker-compose up --build -d mongodb python-backend && cd backend/public-api && npm run dev",
+    "deploy:frontend": "cd frontend/ && npm run start"
```
Comment (Fr3574) [inline package.json:12]: From an end-users perspective, do we need those?

Comment (Corgam) [reply package.json:12]: No, just Theo wanted these back

Comment (thdrbrkw) [reply package.json:12]: Yes, I think its a nice development feature to have

**Review by thdrbrkw — APPROVED** [review]
Looks good, haven't tested it though

**Pull request merged**